spec/sw_indexer_engine_spec.rb

```diff
@@ -9,15 +9,19 @@
 
   describe 'index' do
     context 'for targets that should not be skipped checking' do
-      it 'calls solr delete for the index' do
+      it 'calls solr delete for the index and calls dor-services 856 generation call' do
```

Comment:
added a test and restructured the other existing tests a bit to make it clearer what was being tested